### PR TITLE
Added Tests to Verify MT-SRE Can Connect To Prometheus , Alert Manager Using OpenShift Port-Forwarding

### PR DESCRIPTION
**Pull request by prudhvi-165**

### CODE REVIEW

**Comment by prudhvi-165** [comment]
Continuation of older PR which got closed due to some problems while squashing commits and rebasing [PR Link](https://github.com/red-hat-data-services/ods-ci/pull/335)

**Comment by prudhvi-165** [comment]
Jenkins Link: https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/818/
Result: Passed

File: tests/Resources/Common.robot [hunk 1]
```diff
@@ -83,3 +84,14 @@ Get Cluster Name By Cluster ID
...
+Wait Until HTTP Status Code Is
+    [Arguments]  ${url}   ${expected_status_code}=200  ${retry}=1m   ${retry_interval}=15s
+    Wait Until Keyword Succeeds    ${retry}   ${retry_interval}
+    ...    Check HTTP Status Code    ${url}    ${expected_status_code}
+
+Check HTTP Status Code
+    [Arguments]  ${link_to_check}  ${expected}=200
+    ${response}=    RequestsLibrary.GET  ${link_to_check}   expected_status=any
+    Run Keyword And Continue On Failure  Status Should Be  ${expected}
+    [Return]  ${response.status_code}
```
Comment (lugi0) [inline tests/Resources/Common.robot:97]: Missing [Documentation]

File: tests/Resources/ODS.robot [hunk 2]
```diff
@@ -202,13 +202,23 @@ Verify Default Access Groups Settings
         Verify RHODS Groups Config Map Contains Expected Values   &{exp_values}
     END
 
-Disable Access To Grafana Using OpenShift Port Forwarding
+Disable Access Using OpenShift Port Forwarding
     [Documentation]   Kill process running in background based on Id
-    [Arguments]  ${PROC}
-    Terminate Process   ${PROC}
+    [Arguments]  ${port_forwarding_process}
+    Terminate Process   ${port_forwarding_process}
```
Comment (lugi0) [inline tests/Resources/ODS.robot:208]: This keyword seems to be general but then the names are oddly specific.
I also don't understand if it's supposed to be killing a process on the machine running the test or somewhere in the cluster.

File: tests/Resources/ODS.robot [hunk 3]
```diff
@@ -202,13 +202,23 @@ Verify Default Access Groups Settings
...
+
+Enable Access To Prometheus Using OpenShift Port Forwarding
+    [Documentation]  Enable Access to Prometheus Using OpenShift Port-Forwarding
+    ${port_forwarding_process} =  Start Process   oc -n redhat-ods-monitoring port-forward $(oc get pods -n redhat-ods-monitoring | grep prometheus | awk '{print $1}') 9090  shell=True  # robocop: disable
+    [Return]    ${port_forwarding_process}
+
+Enable Access To Alert Manager Using OpenShift Port Forwarding
+    [Documentation]  Enable Access to Alert Manager Using OpenShift Port-Forwarding
+    ${port_forwarding_process} =  Start Process   oc -n redhat-ods-monitoring port-forward $(oc get pods -n redhat-ods-monitoring | grep prometheus | awk '{print $1}') 9093   shell=True  # robocop: disable
+    [Return]    ${port_forwarding_process}
```
Comment (lugi0) [inline tests/Resources/ODS.robot:223]: Not a big deal but I'd suggest using different names for these variables, we risk running into hard to debug issues if the values are getting overwritten somewhere

**Review by lugi0 — CHANGES_REQUESTED** [review]

File: tests/Resources/Common.robot (hunk 1 above)
Comment (prudhvi-165) [reply tests/Resources/Common.robot]: Updated

File: tests/Resources/ODS.robot (hunk 2 above)
Comment (prudhvi-165) [reply tests/Resources/ODS.robot]: Updated

File: tests/Resources/ODS.robot (hunk 3 above)
Comment (prudhvi-165) [reply tests/Resources/ODS.robot]: Updated.

File: tests/Resources/ODS.robot [hunk 4]
```diff
@@ -202,13 +202,23 @@ Verify Default Access Groups Settings
         Verify RHODS Groups Config Map Contains Expected Values   &{exp_values}
     END
 
-Disable Access To Grafana Using OpenShift Port Forwarding
+Kill Process
     [Documentation]   Kill process running in background based on Id
-    [Arguments]  ${PROC}
-    Terminate Process   ${PROC}
+    [Arguments]  ${proc}
+    Terminate Process   ${proc}
```
Comment (lugi0) [inline tests/Resources/ODS.robot:208]: Actually, I'm not sure it makes sense to have this as a keyword at all, you're simply wrapping the call to `Terminate Process` in the `Kill Process` keyword.

Comment (prudhvi-165) [reply tests/Resources/ODS.robot]: Updated

**Review by lugi0 — APPROVED** [review]
lgtm

**Comment by prudhvi-165** [comment]
Jenkins link: https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/833/

**Review by jgarciao — APPROVED** [review]
lgtm

**Pull request merged**